src/routes/AuthRoute.ts

```diff
@@ -0,0 +1,8 @@
+import express from 'express'
+import { verifyTokenController } from '../controllers/AuthController'
```

Comment:
nah, we will follow a different pattern. call it here AuthController and below `AuthController.verifyToken`

Reply:
This makes sense! See, the reason I added `controller` to `verifyToknControllar` is that I had another `verifyToken` function. Kinda mixed names situation.